config/install/README.md

```diff
@@ -1,101 +1,326 @@
 
 ```bash
 kubectl get deployments -n kuadrant-system
 
-
+## Output
 # NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
 # authorino-operator                      1/1     1            1           83m
 # dns-operator-controller-manager         1/1     1            1           83m
 # kuadrant-console-plugin                 1/1     1            1           83m
 # kuadrant-operator-controller-manager    1/1     1            1           83m
 # limitador-operator-controller-manager   1/1     1            1           83m
+
 ```
 
+You can also view the subscription for information:
 
+```bash
+kubectl get subscription -n kuadrant-system -o=yaml
+
+```
+
+### Install the operand components
+
+Kuadrant has 2 additional operand components that it manages (Authorino that provides data plane auth and Limitador that provides data plane rate limiting). To set these up lets add a new `kustomization.yaml` in a new sub directory. We will re-use this later for further configuration. We do this as a separate step as we want to have the operators installed and in place first.
+
+Add the following to your local directory.  For the purpose of this doc, we will use: `kuadrant/configure/`.
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - https://github.com/Kuadrant/kuadrant-operator//config/install/configure/standard?ref=v1.0.1 #change this version as needed (see https://github.com/Kuadrant/kuadrant-operator/releases)
+
+```
+
+Lets apply this to your cluster:
 
 ```bash
-kubectl get deployments -n gateway-system
 
+kubectl apply -k <kuadrant/configure>
 
-# NAME            READY   UP-TO-DATE   AVAILABLE   AGE
-# istiod          1/1     1            1           61s
-# sail-operator   1/1     1            1           81m
 ```
 
-## Configure the installation
+### Verify kuadrant is installed and ready:
+
+```bash
+kubectl get kuadrant kuadrant -n kuadrant-system -o=wide
+
+# NAME       STATUS   AGE
+# kuadrant   Ready    109s
 
-### TLS and DNS integration
+```
 
+You should see the condition with type `Ready` with a message of `kuadrant is ready`. 
 
-Create the `$CLOUD_PROVIDER-credentials.env file` in the cloud provider directory `config/install/configure/$CLOUD_PROVIDER.` e.g. `aws-credentials.env` in the `config/install/configure/aws` directory. Apply the configuration for the desired cloud provider. Example AWS
+
+### Verify Istio is configured and ready:
+
+```bash
+kubectl get istio -n gateway-system
+
+#sample output
+# NAME      REVISIONS   READY   IN USE   ACTIVE REVISION   VERSION   AGE
+# default   1           1       1        Healthy           v1.23.0   3d22h
+```
+
+
+
+At this point Kuadrant is installed and ready to be used as is Istio as the gateway provider. This means AuthPolicy and RateLimitPolicy can now be configured and used to protect any Gateways you create. 
+
+
+## Configure a DNS provider credentials and a Certificate Issuer
```

Comment:
not needed. Is optional

Reply:
who do we want the audience of this doc to be? what are we trying to show or teach people? 

My thinking is if we document different parts of Kuadrant as optional even tho they technically might be, its going to be a pain for us in the long run with keeping things up to date, making docs follow the same installation etc, are we just overcomplicating things?  

Should we just show how to install kuadrant as a whole and if they don't want to use a certain aspect then so be it. We could show in the architecture what components come out of the box and what are optional. 

side note: I think we need to have a convo as a team on how we want to portray the project in our docs as we are currently writing in all different styles and ways. I'm currently writing a doc (doception 😂 ) to suggest a standard and have a kick off point for conversation. 

This install guide is a bit of a double whammy because we also need a convo on what installation methods do we want to stand behind. we have many different types and different docs on how to do it and duplicate docs that go old and obsolete